chart/README.md

```diff
@@ -0,0 +1,8 @@
+# Feedbackssystem Helm Chart
+
```

Comment:
It would be nice if the required dependencies were listed in the manual. And some instruction steps are missing